lib/Github/Api/Organization/Teams.php

```diff
@@ -83,8 +83,12 @@ public function repository($team, $username, $repository)
         return $this->get('teams/'.rawurlencode($team).'/repos/'.rawurlencode($username).'/'.rawurlencode($repository));
     }
 
-    public function addRepository($team, $username, $repository)
+    public function addRepository($team, $username, $repository, array $params)
     {
+        if (isset($params['permission']) && !in_array($params['permission'], array('pull', 'push', 'admin'))) {
+            $params['permission'] = 'pull';
```

Comment:
I am literally following the standard way this has been done not 20 lines above. Would you like me to change that one as well?